docs/en/ingest-management/elastic-agent/running-on-kubernetes-gke-standard-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,170 @@
+[[running-on-kubernetes-gke-standard-managed-by-fleet]]
+= Run {agent} on GKE standard managed by {fleet}
+
+Use {agent} https://www.docker.elastic.co/r/beats/elastic-agent[Docker images] on Kubernetes to
+retrieve cluster metrics.
+
+TIP: Running {ecloud} on Kubernetes? Refer to {eck-ref}/k8s-elastic-agent-fleet.html[Run {elastic-agent} on ECK].
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of {agent} has not yet been
+released, so no Docker image is currently available for this version.
+
+endif::[]
+
+[discrete]
+== Prerequisites
+
+All data_streams with the `state_` prefix require `hosts` field pointing to `kube-state-metrics`
+service within the cluster.
+
+Kube-state-metrics is not deployed by default in Kubernetes GKE. In order to install kube-state-metrics follow
+the instructions for its deployment are available https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment[here].
+Generally `kube-state-metrics` runs a `Deployment` and is accessible via a service called `kube-state-metrics` on
+`kube-system` namespace, which will be the service to use in our configuration.
+
+
+[discrete]
+== Kubernetes deploy manifests
```

Comment:
I think all this section can be removed and go to elastic-agent doc. So to keep it common for all 

In here I would like something like list of steps until someone can have GKE monitored with our solution

1. Login to GKE and and retrieve credentials for specific GKE cluster 
<gcloud .....> example command
2. Verify your cluster locally by running kubectl command 
3. Login to kibana and go to specific menu and 
3a. Install Integration
3b Create Agent Policy
3c Bind Policy to Agent
4. Apply  manifest to your cluster
5. Verify that your agent is running --> Can we show logs of the agent pod?
6. Verify that your policy is applied  --> Any indication there? Probably from UI
7. Verify logs/ metrics in your kibana discovery 
7a. Explain where logs come from and where metrics come from and should be under which data-view

Reply:
This steps are already part of the original "Run Agent on k8s" documentation page. Is this what you mean by "elastic-agent doc"?

To my mind I wouldn't remove the steps and redirect to another basic page since here we need a page that would be just enough for the GKE users without the need to jump to other pages and combine information. With this the experience is more simple and robust -> "I want to run Agent on GKE I follow the whole page and that works".

Also I don't like to have different verification steps for GKE compared to basic k8s.
Let me know what you think.

Reply:
Makes sense. Then lets update cetral main page